### PR TITLE
adding auto migrations and models

### PR DESCRIPTION
**Pull request by Peleg**
now just importing the db package should get us access to the db w/o having to reconnect

### CODE REVIEW

File: app/db/db.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package db
+
+import "github.com/jinzhu/gorm"
+import _ "github.com/lib/pq"
```
Comment (pathouse) [inline app/db/db.go:4]: ?

Comment (pathouse) [inline app/db/db.go:4]: ok so local alias. do you know what it means when the local alias is `_`, does that directly import package stuff without any namespace at all?

Comment (Peleg) [inline app/db/db.go:4]: tbh just moved this from the app/init.go but look at this: http://golang.org/doc/effective_go.html#blank
In this case, gorm needs the postgres to go through its init function, but we're not using the pq variable directly so w/o the _, itll bitch about an unused var and wont compile

File: app/db/db.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  if err != nil {
+    panic(fmt.Sprintf("Got error when connect database, the error is '%v'", err))
+  }
+
+  // defaults
+  DB.DB().SetMaxIdleConns(10)
+  DB.DB().SetMaxOpenConns(100)
+  DB.DB().Ping()
+
+  for _, model := range []interface{}{models.User{}, models.Post{}} {
```
Comment (ScotterC) [inline app/db/db.go:24]: can we create a collection of models somewhere?  this needs to be done with all of them right?

**Comment by ScotterC** [comment]
Cool!  @pathouse any idea of how we can get a collection of all models we want auto migrated?

**Pull request merged**